cassandra-plugins/src/test/java/io/cdap/plugin/test/ETLCassandraTest.java

```diff
@@ -87,9 +86,6 @@ public class ETLCassandraTest extends HydratorTestBase {
   @ClassRule
   public static TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-  @ClassRule
-  public static final TestConfiguration CONFIG = new TestConfiguration("explore.enabled", false);
```

Comment:
No need to remove CONFIG. We should just be removing the config passed to it. Please make similar change in all places.

Reply:
Actually, in the PR mentioned in description, Vitalli suggested that if there are no custom settings in the CONFIG, the classrule is not needed. So I have removed similar references from cdap as well.

Reply:
Got it 👍